docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
             REDIS_URL: "redis://redis:6379/"
             PUPPETEER_WS_ENDPOINT: "ws://browserless:3000" # marked
         healthcheck:
-            test: ["CMD", "curl", "-f", "http://localhost:1200/healthz"]
+            test: ["CMD", "curl", "-f", "http://localhost:1200/healthz${ACCESS_TOKEN:+?key=${ACCESS_TOKEN}}"]
```

Comment:
> Update your own docker compose file instead since this one is used as a starting point for customisation and should only include the most essential configs

_Originally posted by @TonyRL in https://github.com/DIYgod/RSSHub/pull/17144#discussion_r1805161276_